diff --git a/node_modules/@uniswap/v3-periphery/contracts/libraries/BytesLib.sol b/node_modules/@uniswap/v3-periphery/contracts/libraries/BytesLib.sol
index ca1ae84..e063c76 100644
--- a/node_modules/@uniswap/v3-periphery/contracts/libraries/BytesLib.sol
+++ b/node_modules/@uniswap/v3-periphery/contracts/libraries/BytesLib.sol
@@ -6,7 +6,7 @@
  * @dev Bytes tightly packed arrays utility library for ethereum contracts written in Solidity.
  *      The library lets you concatenate, slice and type cast bytes arrays both in memory and storage.
  */
-pragma solidity >=0.5.0 <0.8.0;
+pragma solidity >=0.5.0;
 
 library BytesLib {
     function slice(
